content/strategy-goals/strategy/code-graph/search/product.md

```diff
@@ -10,37 +10,71 @@ Quicklinks:
 - [Search Product backlog](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Ateam%2Fsearch-product/)
 - [Roadmap](https://github.com/orgs/sourcegraph/projects/214/views/34?filterQuery=owning-org%3A%22Code+Graph%22+type%3ARoadmap+owning-team%3A%22Search+product%22)
 
+## Goals
+
+- Continue to differentiate the core platform beyond code search. Search will always power our platform, but to remain competitive we need to become more than the world's best code search
+
+- Provide compelling solutions for the most pressing use cases of our customers
+
+- Maintain room for experimentation and exploratory work that creates meaningful business value
+
 ## Where we are now
 
-Sourcegraph is the leading code search product in the market, used by tens of thousands of developers across enterprises of all sizes. It has advanced, novel search capabilities, such as [structural search](https://learn.sourcegraph.com/how-to-use-structural-search-in-sourcegraph) and [diff/commit search](https://sourcegraph.com/notebooks/Tm90ZWJvb2s6MTI=), as well as capabilities that extend search, such as [Code monitors](https://docs.sourcegraph.com/code_monitoring) and [Notebooks](https://sourcegraph.com/notebooks?tab=explore).
+Sourcegraph is the leading code intelligence platform in the market, used by tens of thousands of developers across enterprises of all sizes. The engine powering the platform is Code Search, which has advanced, novel search capabilities, such as [structural search](https://learn.sourcegraph.com/how-to-use-structural-search-in-sourcegraph) and [diff/commit search](https://sourcegraph.com/notebooks/Tm90ZWJvb2s6MTI=).
 
-<!-- TODO: fix up docs link to go to Notebooks when docs PR is merged -->
+The Search Product team has the dual mission of increasing the value of the core platform by developing best-in-class capabilities for use by all product teams, and building differentiated products and features of its own.
```

Comment:
Does this describe something that the search product team is doing in particular? As I read it, any software team making anything would have the same mission (make our product better by making good features that people use).